### PR TITLE
Unit tests -> Do unit tests for GetPopularMoviesUseCase

### PR DESCRIPTION
**Pull request by Nataniel-Antosik**

### CODE REVIEW

File: app/src/test/java/com/example/benchproject/domain/popular/movies/usecase/GetPopularMoviesUseCaseTest.kt [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    private val movieListModel = Result.success(
+        listOf(
+            MovieModel(10001, "Test1", 5.4, "2022-06-23", "https:/something1.com"),
+            MovieModel(10002, "Test2", 6.4, "2022-06-22", "https:/something2.com"),
+            MovieModel(10003, "Test3", 3.4, "2022-06-21", "https:/something3.com")
+        )
+    )
+    private val badResponseFromRepository =
+        Result.failure<List<MovieModel>>(
+            Throwable(
```
Comment (BorysChajdas) [inline app/src/test/java/com/example/benchproject/domain/popular/movies/usecase/GetPopularMoviesUseCaseTest.kt]: New line redundant here, message could be in one line

File: app/src/test/java/com/example/benchproject/domain/popular/movies/usecase/GetPopularMoviesUseCaseTest.kt [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                "Unable to resolve host \"api.themoviedb.org\": No address associated with hostname"
+            )
+        )
+    private val popularMoviesRepository: PopularMoviesRepository = mockk()
+    private val tested = GetPopularMoviesUseCase(popularMoviesRepository)
```
Comment (BorysChajdas) [inline app/src/test/java/com/example/benchproject/domain/popular/movies/usecase/GetPopularMoviesUseCaseTest.kt:26]: Add new line here ;)

**Review by BorysChajdas — APPROVED** [review]

File: app/src/test/java/com/example/benchproject/domain/popular/movies/usecase/GetPopularMoviesUseCaseTest.kt (hunk 1 above)
Comment (Nataniel-Antosik) [reply app/src/test/java/com/example/benchproject/domain/popular/movies/usecase/GetPopularMoviesUseCaseTest.kt]: Ok I deleted this new line and now it looks like 
```
private val badResponseFromRepository =
        Result.failure<List<MovieModel>>(
            Throwable("Unable to resolve host \"api.themoviedb.org\": No address associated with hostname")
        )
```

File: app/src/test/java/com/example/benchproject/domain/popular/movies/usecase/GetPopularMoviesUseCaseTest.kt (hunk 2 above)
Comment (Nataniel-Antosik) [reply app/src/test/java/com/example/benchproject/domain/popular/movies/usecase/GetPopularMoviesUseCaseTest.kt:26]: I added new line :D

**Pull request merged**